projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -73,6 +74,53 @@ const HomeScreenHeader = withNavigation(
     },
 )
 
+const IssueListFooter = ({ navigation }: NavigationInjectedProps) => {
+    const isUsingProdDevtools = useIsUsingProdDevtools()
+    const { setIssueId } = useIssueSummary()
+
+    return (
+        <View
+            style={{
+                padding: metrics.horizontal,
```

Comment:
Strongly prefer moving inline styles to a separate const. Better separation of concerns.

Reply:
This occurs in multiple places